### PR TITLE
Show room number on meeting calendar

### PR DESCRIPTION
**Pull request by YBenPan**

### CODE REVIEW

File: src/pages/meetings.js [hunk 1]
```diff
@@ -77,7 +80,9 @@ const Meetings = () => {
 							?.filter(meeting => meeting.privacy !== "private" || audits || user?.isFaculty)
 							.map(meeting => {
 								const newMeeting = { ...meeting };
-								newMeeting.title = meeting.organization.name + " - " + meeting.title;
+								const meetingRoom =
+									typeof meeting.rooms[0] === "undefined" ? "" : `: Room ${meeting.rooms[0].name}`;
```
Comment (vityavv) [inline src/pages/meetings.js:84]: should not be in a string? anyway test again make sure I'm not making stuff up

File: src/pages/org/meetings.js [hunk 2]
```diff
@@ -62,7 +62,12 @@ export default function Meetings() {
 						start: "title",
 						end: "dayGridMonth listMonth prev next"
 					}}
-					events={org.meetings}
+					events={org?.meetings.map(meeting => {
+						const newMeeting = { ...meeting };
+						newMeeting.title +=
+							typeof meeting.rooms[0] === "undefined" ? "" : `: Room ${meeting.rooms[0].name}`;
```
Comment (vityavv) [inline src/pages/org/meetings.js:68]: same thing here

**Review by vityavv — CHANGES_REQUESTED** [review]

File: src/pages/meetings.js (hunk 1 above)
Comment (YBenPan) [reply src/pages/meetings.js:84]: I did `console.log(meeting.rooms[0]);` and got this:
image

so `meeting.rooms[0].name` is indeed a string if `meeting.rooms[0]` is defined.

Comment (YBenPan) [reply src/pages/meetings.js:84]: And this is an image of what the calendar looks like with this PR: 
image
image

**Pull request merged**